quizbizz/src/util/toasts/toasts.css

```diff
@@ -0,0 +1,18 @@
+.toasts-container{
+    z-index: 1000;
```

Comment:
Suggestion: You should not use that higher z-index value.

Reply:
But i want it to come over all the components that is why i have used it like that